changelogs/fragments/4039-cluster-container-wait.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - "Adds a wait period when creating clustered containers for when the task is still Running and not immediately jump to success. This would error out when the task was still running."
```

The change itself looks more like:
```suggestion
  - "lxd inventory plugin, lxd_container module, lxd_profile module - adds a wait period when creating clustered containers for when the task is still running and not immediately jump to success. This would error out when the task was still running (https://github.com/ansible-collections/community.general/pull/4039)."
```
The `do()` method is used all over the place in all these functions.

---

Added